.changeset/plenty-donkeys-draw.md

```diff
@@ -0,0 +1,6 @@
+---
+'@keystonejs/fields': patch
+'@keystonejs/keystone': patch
```

Comment:
`patch` level change because the value was hard coded to `undefined` before anyway, so now it will still be `undefined`:

```javascript
const { existingItem } = { foo: 'bar' };
console.log(existingItem);
// undefined
```